.github/actions/build-sof-doc.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+# build doc from sof
+git clone https://github.com/thesofproject/sof
+cmake -GNinja -S sof/doc -B _build_doxy
+
+# TODO: change the (bad) default value for SOF_DOC_BUILD in
+# sof-docs/Makefile and remove this command line override
+# build sof-doc
+make html VERBOSE=1 SOF_DOC_BUILD=_build_doxy
+du -shc _build*/*
```

Comment:
Thanks for moving this to a script

Reply:
This is just a fancier `ls` to show what was just built and make sure it looks roughly as expected. This will show any mistake and or confusion between with `_build`, `_build_doxy`, `SOF_DOC_BUILD` etc.